src/LightGBM.jl

```diff
@@ -16,21 +16,25 @@ struct LibraryNotFoundError <: Exception
 end
 
 
-function find_library(library_name::String, custom_paths::Vector{String})
+function find_library(library_names::Vector{String}, custom_paths::Vector{String})
 
-    # Search system filedirs first, returns empty string if not found
-    libpath = Libdl.find_library(library_name)
+    # If any of library_names is not an absolute path name, then the paths in the DL_LOAD_PATH and 
+    # system load path are searched. Returns empty string if not found.
+    libpath = Libdl.find_library(library_names)
 
-    if libpath == ""
-        # try specified paths
-        @info("$(library_name) not found in system dirs, trying fallback")
-        libpath = Libdl.find_library(library_name, custom_paths)
+    if libpath != ""
+        @info("$(library_names) found in `DL_LOAD_PATH`, or system library paths $(ENV["PATH"])!")
```

Comment:
`ENV["PATH"]` is only the system path for link libraries on windows. This doesn't apply in linux or mac. See the README